contracts/validator-registration-contract/validator_registration.go

```diff
@@ -231,21 +231,21 @@ func (_ValidatorRegistration *ValidatorRegistrationCallerSession) UsedPubkey(arg
 // Deposit is a paid mutator transaction binding the contract method 0x881d2135.
 //
 // Solidity: function deposit(_pubkey bytes32, _withdrawalShardID uint256, _withdrawalAddressbytes32 address, _randaoCommitment bytes32) returns()
-func (_ValidatorRegistration *ValidatorRegistrationTransactor) Deposit(opts *bind.TransactOpts, _pubkey [32]byte, _withdrawalShardID *big.Int, _withdrawalAddressbytes32 common.Address, _randaoCommitment [32]byte) (*types.Transaction, error) {
```

Comment:
Did you regenerate `validator_registration.go` or manually changed it `[48]byte`?

Weird it still says `function deposit(_pubkey bytes32,...` at the top